netebpfext/net_ebpf_ext.h

```diff
@@ -18,8 +18,30 @@
 
 #include <ntddk.h>
 
+#define INITGUID
+
+#pragma warning(push)
+#pragma warning(disable : 4201) // unnamed struct/union
+#include <fwpsk.h>
+#pragma warning(pop)
+
+#include <fwpmk.h>
```

Comment:
to verify: this is out of order... does compilation fail if this is included before fwpsk.h?

Reply:
fixed in #520